lib/rules/font-weight-notation/__tests__/index.mjs

```diff
@@ -5,6 +5,7 @@ testRule({
 	ruleName,
 	config: ['numeric'],
 	fix: true,
+	computeEditInfo: true,
```

Comment:
questions

Is this wrong?
i.e. the documentation will state that we can't have both `fix` and `computeEditInfo` both set to `true`
I am talking about the option/flag.

Do we allow it for `testRule`?

Reply:
I think it's fine because testRule runs lint twice when fix is specified.
https://github.com/stylelint/jest-preset-stylelint/blob/main/getTestRule.js#L51-L57

@romainmenke 
Please let me know if my understanding is incorrect.

Reply:
I would avoid doing this.
Better to separate these test cases.

I didn't actually know that it was possible to combine them as such in tests :D

Reply:
I changed my mind about this and suggest adding test coverage for this instead: https://github.com/stylelint/jest-preset-stylelint/pull/142

Reply:
Once you have come to a decision, you may resolve this conversation.
i.e. it's not a blocker anymore now that it's gonna be clarified